cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-state-during-navigation.cy.js

```diff
@@ -0,0 +1,81 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+
+const testData = {};
+const validHoldingUUIDsFileName = `SearchHoldingUUIDs_${getRandomPostfix()}.csv`;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+  instanceId: '',
+  holdingUUID: '',
+  holdingHRID: '',
```

Comment:
do you need these initialized here? instanceId: '',
  holdingUUID: '',
  holdingHRID: '',

Reply:
Yes, it needs me!